libarchive/archive_write_set_format_iso9660.c

```diff
@@ -2553,7 +2553,7 @@ set_date_time(unsigned char *p, time_t t)
 static void
 set_date_time_null(unsigned char *p)
 {
-	memset(p, '0', 16);
+	memset(p, (int)'0', 16);
```

Comment:
The warning was as follows: <pre> 1017786 Memset fill value of '0'

Nominate Defect...
gift  gift  gift
The buffer will be filled with ASCII character '0' instead of actual zero bytes.
In set_date_time_null: A memset fill value of ASCII character '0' is likely intended to be 0 (CWE-665)</pre>

It's warning against valid concerns (noob programmers who don't know how C works might use `'0'` out of ignorance when they meant `0`..

I could ignore the issue in the FreeBSD Coverity runs, but then everyone who runs Coverity against libarchive would need to do the same (NetBSD, Linux variants, etc).